descriptions/api.github.com/api.github.com.2022-11-28.json

```diff
@@ -13956,6 +13956,424 @@
+                  "target": {
+                    "type": "string",
+                    "description": "The target of the ruleset.",
+                    "enum": [
+                      "branch",
+                      "tag"
+                    ]
+                  },
+                  "enforcement": {
+                    "$ref": "#/components/schemas/repository-rule-enforcement"
+                  },
+                  "bypass_actors": {
+                    "type": "array",
+                    "description": "The actors that can bypass the rules in this ruleset",
+                    "items": {
+                      "$ref": "#/components/schemas/repository-ruleset-bypass-actor"
+                    }
+                  },
+                  "conditions": {
+                    "$ref": "#/components/schemas/org-ruleset-conditions"
+                  },
+                  "rules": {
+                    "type": "array",
+                    "description": "An array of rules within the ruleset.",
+                    "items": {
+                      "$ref": "#/components/schemas/repository-rule"
+                    }
+                  }
+                },
+                "required": [
+                  "name",
+                  "enforcement"
+                ]
+              },
+              "examples": {
+                "default": {
+                  "value": {
+                    "id": 21,
+                    "name": "super cool ruleset",
+                    "target": "branch",
+                    "enforcement": "enabled",
+                    "bypass_actors": [
+                      {
+                        "actor_id": 234,
+                        "actor_type": "Team"
+                      }
+                    ],
+                    "conditions": {
+                      "ref_name": {
+                        "include": [
+                          "refs/heads/main",
+                          "refs/heads/master"
+                        ],
+                        "exclude": [
+                          "refs/heads/dev*"
+                        ]
+                      }
+                    },
+                    "rules": [
+                      {
+                        "type": "commit_author_email_pattern",
+                        "parameters": {
+                          "operator": "contains",
+                          "pattern": "github"
+                        }
+                      }
+                    ]
+                  }
+                }
+              }
+            }
+          }
+        },
+        "responses": {
+          "201": {
+            "description": "Response",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/repository-ruleset"
+                },
+                "examples": {
+                  "default": {
+                    "$ref": "#/components/examples/org-ruleset"
```

Comment:
Is this accurate? pointing to an `org-ruleset` example for services that return a `repository-ruleset`?

Reply:
So `repository-ruleset` is the shared schema
`org-ruleset` is just an example of a `repository-ruleset` configured for an org.